### ISSUE TITLE
v0.4.0 Build Fails due to AFL

### ISSUE THREAD

**Issue by EliSauder** [kristoff-it/superhtml#21]
The new v0.4.0 build fails

The line that fails is 117 of build.zig (not in the error message, but the many `std.debug.print("here#\n", .{})` say it is this line 🤣).
https://github.com/kristoff-it/superhtml/blob/4c20f0b2570e99b43bd85783fb306411e775e56e/build.zig#L117

```
unable to spawn the following command: FileNotFound
llvm-config --version
error: the following build command failed with exit code 1:
/Users/esauder/source/repos/personal/superhtml/.zig-cache/o/d1e56a8d062b567fb6786b74e3e02a26/build /Users/esauder/.zvm/0.13.0/zig /Users/esauder/source/repos/personal/superhtml /Users/esauder/source/repos/personal/superhtml/.zig-cache /Users/esauder/.cache/zig --seed 0xcb9dae9f -Z28ec7a187d155df7
```

This can also be seen in the ci:
<img width="946" alt="image" src="https://github.com/user-attachments/assets/8523268a-a7a1-4227-bb02-e59042497f9e">

**Comment by EliSauder** [comment]
I had time to get my path variable working (I have llvm installed, but bc apple, I normally reference the path directly). Even after getting llvm-config, I get this error when building.

<img width="909" alt="image" src="https://github.com/user-attachments/assets/8bf2d0e1-bc25-4ed1-8a01-64e6382eae87">

**Comment by EliSauder** [comment]
Adding:　`pub const version = build_options.version;` to the cli.zig file works, though idk if that is what you want with the project.

**Comment by kristoff-it** [comment]
The overly aggressive dependency on llvm-config has been fixed, let's continue the discussion in #22
